includes/wsuwp-shortcode-fundselector.js

```diff
@@ -297,19 +302,29 @@ jQuery(document).ready(function($) {
 		});
 		loadPriorities($("#priorities"), "idonate_priorities", "idonate_priorities");
 	}
-	else if (wpData.area && wpData.cat) {
+	else if ((wpData.cat && !wpData.area && !wpData.unit_designation) || (wpData.cat && wpData.area && !wpData.unit_designation) || (wpData.cat && wpData.area && wpData.unit_designation)) {
```

Comment:
I believe this can be simplified to `(wpData.cat || (wpData.cat && wpData.area) || (wpData.cat && wpData.area && wpData.unit_designation))` and still accomplish the same thing

Reply:
There's only one case that yours doesn't work the same as mine, and I suppose it just depends what we want the result to be. For mine, if the category and the fund is specified, it populates the fund in the priorities since it doesn't know what area to populate. For yours, it shows the category but doesn't populate the fund anywhere. It just comes down to which one you want to prioritize.